### DISCUSSION TITLE
Highlight, Copy, and Paste Multiple Code Notes into the Achievement Editor

### DISCUSSION THREAD

**Discussion by Gollawiz** [RetroAchievements/RAIntegration#958] [Ideas]
Simple quality of life feature; but when doing large addsource chains you must click each address one at a time and manually add them by pressing the 'new' button in the achievement editor. Would it be possible to change this where we can highlight multiple addresses that are bookmarked or in the code note list and have them pasted all together in the editor? This would reduce time taken to do long chains of code as well as reduce accidentally repeating an address / leaving one out from your desired total.
![codenotes](https://user-images.githubusercontent.com/106358163/213242237-9a28d73d-b4c2-4470-b77c-cd61e52a2635.png)

Example of how many I have to work with on one of my current claims.

**Comment by redwizard42** [comment]
I have tried to do this very thing about 100+ times. I would love this.
